packages/format-library/src/image/index.js

```diff
@@ -73,6 +113,50 @@ export const image = {
 							return null;
 						} }
 					/> }
+					{ isActive && <PositionedAtSelection key={ key }>
+						<Popover
+							position="bottom center"
+							focusOnMount={ false }
+						>
+							{ // Disable reason: KeyPress must be suppressed so the block doesn't hide the toolbar
+							/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */ }
+							<form
```

Comment:
Should we move some of this markup into its own component? `ImageEdit` is getting rather large.